src/broadcast.js

```diff
@@ -0,0 +1,15 @@
+// send new score to API
+
+export default async function broadcast(raw) {
+  const myHeaders = new Headers();
+  myHeaders.append('Content-Type', 'application/json');
+  const requestOptions = {
+    method: 'POST',
+    headers: myHeaders,
+    body: raw,
+    redirect: 'follow',
+  };
+  const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/dV4aWISG7S2Gi5e0alOq/scores';
+  const request = new Request(requestURL);
+  await fetch(request, requestOptions);
+}
```

Comment:
- [ ] Well done writing this function and it's all working fine but the issue is that one of the requirements in this milestone is to use ES6 arrow functions instead of the regular `function` keyword. so kindly consider refactoring it.
project requirement
  -  Use arrow functions instead of the function keyword.